### DISCUSSION TITLE
What's new in Budibase: September 2023

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#11949] [Announcements]
Hey Buddies 👋

The September release has landed :tada:

New features include :sparkles:
:arrows_clockwise:  Retain user & table data when overriding applications
:bust_in_silhouette:  User column type
:calendar:  Date-only field type
:bullettrain_front:  Quick access to published apps (must see :eyes: )
... and more!

You can learn about all the new features and improvements in the release post:
https://budibase.com/blog/updates/2023/september/

**Comment by andyburgessmd** [comment]
I think the user column feature in this release has been badly managed and I think there are a number of us users struggling to get it to work in our applications. Parts of it feel like they are untested and there are parts of it missing which makes existing applications unusable.
We need tools/methods to migrate users and these have not been considered, for example I have a application table with ~900 relationships per user and 10 users, am I supposed to manually reassign 9000 records to the new user type?
